sdk/formrecognizer/azure-ai-formrecognizer/tests/test_invoice.py

```diff
@@ -295,8 +295,8 @@ def test_invoice_multipage_pdf(self, client):
 
     @FormRecognizerPreparer()
     @GlobalClientPreparer()
-    def test_invoice_pdf_include_field_elements(self, client):
-        with open(self.invoice_pdf, "rb") as fd:
+    def test_invoice_jpg_include_field_elements(self, client):
```

Comment:
out of curiosity, why are you changing pdf tests to jpg?

Reply:
Switching to the sample that the service team provided which actually returns all the fields for invoices. It's a jpg; we still have tests that test the pdf form.